source/features/deep-blame.tsx

```diff
@@ -44,13 +44,16 @@ const getPullRequestBlameCommit = mem(async (commit: string, prNumber: number):
 	return false;
 });
 
-function getCommitHash(commit: HTMLElement): string {
-	return (commit.nextElementSibling as HTMLAnchorElement).href.split('/').pop()!;
+function getCommitHash(commit: HTMLLinkElement): string {
```

Comment:
```suggestion
function getCommitHash(commit: HTMLAnchorElement): string {
```

`HTMLLinkElement` is `<link>`